### ISSUE TITLE
Add useful repr for `Configure`

### ISSUE THREAD

**Issue by wtbarnes** [rice-solar-physics/pydrad#159]
Looking at the config parameters as just a dictionary is often difficult because it is so long. It would be useful to add a string repr to the `Configure` object that displays the configuration dictionary in nice format. For example, this could be a set of separate tables, one for each section of the configuration. This could be done with `astropy.table` or `pandas`.

**Comment by sjbradshaw** [comment]
Fair enough. If you want to modify pydrad accordingly then go ahead. Config tools for the code can exist independently. I'll just need to make sure I keep the community up to date with any new parameters I introduce.Sent from my Galaxy
-------- Original message --------From: Will Barnes ***@***.***> Date: 4/7/23  9:01 PM  (GMT-06:00) To: rice-solar-physics/pydrad ***@***.***> Cc: Subscribed ***@***.***> Subject: [rice-solar-physics/pydrad] Add useful repr for `Configure` (Issue #159) 
Looking at the config parameters as just a dictionary is often difficult because it is so long. It would be useful to add a string repr to the Configure object that displays the configuration dictionary in nice format. For example, this could be a set of separate tables, one for each section of the configuration. This could be done with astropy.table or pandas.

**Comment by wtbarnes** [comment]
Yes, I've found that the HYDRAD configuration options evolve slowly enough that keeping pace with the changes has not been an issue.
